### PR TITLE
what field is supposed to map to Salesforce’s required field "Bulk Update Record ID" and "Bulk Upsert External ID"

### PR DESCRIPTION
**Pull request by Panandhan22**
### what fields are supposed to map to Salesforce’s required fields for "Bulk Update Record ID" and "Bulk Upsert External ID"?

For "Bulk Update Record ID", please refer to this [Salesforce’s help documentation](https://help.salesforce.com/s/articleView?id=000385008&type=1).
For "Bulk Upsert External ID", please refer to this [Salesforce’s help documentation](https://help.salesforce.com/s/articleView?id=000383278&type=1).

### CODE REVIEW

**Comment by gayuru** [comment]
@Panandhan22 @kdaswani came across this issue and have been banging my head trying to figure it out 🤣 
- Can we use a custom field for the `Bulk Update Record Id` field?

Screenshot 2023-03-23 at 4 04 25 pm

File: src/connections/destinations/catalog/actions-salesforce/index.md [hunk 1]
```diff
@@ -109,3 +109,8 @@ When **Use Salesforce Bulk API** is enabled for your mapping, events are sent to
 
 For bulk `update`, if a record in a batch is missing a Bulk Update Record ID, Segment will still send it to Salesforce. Salesforce will reject the individual record because it will be unable to find a record to update. Other records in the batch that are valid will still be processed. Please note that Segment's Event Delivery tab will show the entire batch as successful as Segment cannot currently break down Event Delivery stats into individual failed/passed events.
 
+### what fields are supposed to map to Salesforce’s required fields for "Bulk Update Record ID" and "Bulk Upsert External ID"?
+
+For "Bulk Update Record ID", please refer to this [Salesforce’s help documentation](https://help.salesforce.com/s/articleView?id=000385008&type=1).
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/actions-salesforce/index.md:114]:
```suggestion
For "Bulk Update Record ID", see [Salesforce’s help documentation](https://help.salesforce.com/s/articleView?id=000385008&type=1){:target="_blank"}.
```

File: src/connections/destinations/catalog/actions-salesforce/index.md [hunk 2]
```diff
@@ -109,3 +109,8 @@ When **Use Salesforce Bulk API** is enabled for your mapping, events are sent to
...
+For "Bulk Upsert External ID", please refer to this [Salesforce’s help documentation](https://help.salesforce.com/s/articleView?id=000383278&type=1).
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/actions-salesforce/index.md:115]:
```suggestion
For "Bulk Upsert External ID", see[Salesforce’s help documentation](https://help.salesforce.com/s/articleView?id=000383278&type=1){:target="_blank"}.
```

File: src/connections/destinations/catalog/actions-salesforce/index.md [hunk 3]
```diff
@@ -109,3 +109,8 @@ When **Use Salesforce Bulk API** is enabled for your mapping, events are sent to
 
 For bulk `update`, if a record in a batch is missing a Bulk Update Record ID, Segment will still send it to Salesforce. Salesforce will reject the individual record because it will be unable to find a record to update. Other records in the batch that are valid will still be processed. Please note that Segment's Event Delivery tab will show the entire batch as successful as Segment cannot currently break down Event Delivery stats into individual failed/passed events.
 
+### what fields are supposed to map to Salesforce’s required fields for "Bulk Update Record ID" and "Bulk Upsert External ID"?
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/actions-salesforce/index.md:112]:
```suggestion
### Which fields are supposed to map to Salesforce’s required fields for "Bulk Update Record ID" and "Bulk Upsert External ID"?
```

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**